lib/Dialect/TritonNvidiaGPU/Transforms/CheckMatmulTwoCTAs.cpp

```diff
@@ -0,0 +1,63 @@
+#include "triton/Dialect/TritonNvidiaGPU/IR/Dialect.h"
+#include "triton/Dialect/TritonNvidiaGPU/Transforms/Passes.h"
+
+#include "mlir/IR/BuiltinAttributes.h"
+#include "mlir/IR/BuiltinOps.h"
+#include "mlir/IR/Diagnostics.h"
+#include "mlir/IR/Visitors.h"
+
+namespace ttng = mlir::triton::nvidia_gpu;
+
+namespace mlir::triton::nvidia_gpu {
+
+#define GEN_PASS_DEF_TRITONNVIDIAGPUCHECKMATMULTWOCTAPASS
+#include "triton/Dialect/TritonNvidiaGPU/Transforms/Passes.h.inc"
+
+namespace {
+
+class TritonNvidiaGPUCheckMatmulTwoCTAPass
```

Comment:
If this pass is intended for the new requirement from CUDA 13, shouldn't we also check non-MMA ops like TMA and tmem_copy for which PTX supports 2CTA mode?